smbprotocol/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "smbprotocol" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 98218617ce75870b5b0ea3951269e5db169b1e86509d09d61b64c0a749a3a1f3
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cryptography >=2.0
+    - python-gssapi >=1.4.1  # [linux]
```

Comment:
Rather than a run dep I think this should probably be a `run_constrained` dep and can relax it to `[not win]`